.gitignore

```diff
@@ -4,5 +4,5 @@ config.js
 npm-debug.log*
 TODO.txt
 dest/
-.env
```

Comment:
IDE must have removed it when I formatted (because I don't have that file in my tree), I'll add it back in.